ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -199,6 +203,16 @@ object Config {
         .action((interval, config) => config.copy(metricsReportingInterval = interval))
         .hidden()
 
+      opt[Boolean]("no-indexer")
+        .optional()
+        .text("Do not start any indexer.")
+        .action((enabled, config) => config.copy(noIndexer = enabled))
+
+      opt[Boolean]("no-ledger-api-server")
+        .optional()
+        .text("Do not start any ledger API server.")
+        .action((enabled, config) => config.copy(noLedgerApiServer = enabled))
```

Comment:
I would expect these options to be at the participant level, not for all participants. I'd make it part of the `--participant` option.